docker-compose.yaml

```diff
@@ -27,3 +27,4 @@ services:
       REDIS_PORT: 6379
       REDIS_PASSWORD: ""
       SERVICE_URI: http://localhost:3000
+      TEST_TOKEN: ${TEST_TOKEN:-0f9ecf32-da18-4e17-8a94-c37732d97489}
```

Comment:
It might be best to not have a default value for this as then it will need to be specifically set in the environment.
Also, can a new line character be added to the file please?

Reply:
Done